best-practices/database/README.md

```diff
@@ -1,4 +1,9 @@
-# DATABASE
+# Databases
+A database is an organized collection of data, generally stored and accessed electronically from a computer system. Where databases are more complex they are often developed using formal design and modeling techniques. 
+
+At Shift3, we generally use T-SQL and NoSQL databases in our projects. More information can be found [here](https://www.altexsoft.com/blog/business/comparing-database-management-systems-mysql-postgresql-mssql-server-mongodb-elasticsearch-and-others/).
```

Comment:
You might want to update the Shift3 reference.

Reply:
I totally missed that. I made the change before the announcement. 

Ill change it, thanks!

Reply:
fixed via: 4267c2a2ecbe5c136f87062bbc278fb541bafb89